mapfishapp/src/main/webapp/app/addons/notes/js/main.js

```diff
@@ -0,0 +1,149 @@
+Ext.namespace("GEOR.Addons");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Ext" is not defined.](https://www.codacy.com/app/georchestra/georchestra/file/2331255335/issues/source?bid=3200181&fileBranchId=3206983#l1)